export/export.go

```diff
@@ -180,105 +196,146 @@ func (s *Service) Import(ctx context.Context, resources *ResourcesForImport) err
 
 	// `applyPvs` should be called after `applyPvcs` finished,
 	// because `applyPvs` look up PersistentVolumeClaim for `Spec.ClaimRef.UID` field.
-	if err := s.applyPvs(ctx, resources, &errgrp); err != nil {
-		return xerrors.Errorf("call applyPvs: %w", err)
+	if err := s.applyPvs(ctx, resources, &errgrp, opts); err != nil {
+		return err
 	}
 	if err := errgrp.Grp.Wait(); err != nil {
 		return xerrors.Errorf("apply PVs: %w", err)
 	}
+	return nil
+}
 
+// Import imports all resources from posted data.
+// (1) Restart scheduler based on the data.
+// (2) Apply each resource.
+//     * If UID is not nil, an error will occur. (try to find existing resource by UID)
+func (s *Service) Import(ctx context.Context, resources *ResourcesForImport, opts ...Option) error {
+	options := options{}
+	for _, o := range opts {
+		o.apply(&options)
+	}
+	if err := s.schedulerService.RestartScheduler(resources.SchedulerConfig); err != nil {
+		return xerrors.Errorf("restart scheduler with imported configuration: %w", err)
+	}
+	if err := s.apply(ctx, resources, options); err != nil {
+		return xerrors.Errorf("import resources all: %w", err)
+	}
 	return nil
 }
 
-func (s *Service) listPods(ctx context.Context, r *ResourcesForExport, eg *util.ErrGroupWithSemaphore) error {
+func (s *Service) listPods(ctx context.Context, r *ResourcesForExport, eg *util.ErrGroupWithSemaphore, opts options) error {
 	if err := eg.Sem.Acquire(ctx, 1); err != nil {
 		return xerrors.Errorf("acquire semaphore: %w", err)
 	}
 	eg.Grp.Go(func() error {
 		defer eg.Sem.Release(1)
 		pods, err := s.podService.List(ctx)
 		if err != nil {
-			return xerrors.Errorf("call list pods: %w", err)
+			if !opts.ignoreErr {
+				return xerrors.Errorf("call list pods: %w", err)
+			}
+			klog.Errorf("failed to call list pods: %v", err)
+			pods = &corev1.PodList{Items: []corev1.Pod{}}
 		}
 		r.Pods = pods.Items
```

Comment:
I added this line.
```go
pods = &corev1.PodList{Items: []corev1.Pod{}}
```

If `s.podService.List(ctx)` return an error, `pods` is going to `nil`.
So, runtime error has occurred in here.